src/core/services/gesture/gesture.js

```diff
@@ -302,7 +302,7 @@ angular.module('material.core')
     onCancel: angular.noop,
     options: {},
 
-    dispatchEvent: dispatchEvent,
+    dispatchEvent: angular.element === window.jQuery ? jQueryDispatchEvent : jqLiteDispatchEvent,
```

Comment:
We shouldn't check for jQuery on `window`, this won't work in all situations.

This would work better:

```js
dispatchEvent: typeof jQuery !== 'undefined' && angular.element === jQuery ? 
  jQueryDispatchEvent : 
  jqLiteDispatchEvent,
```